src/Compilers/Server/VBCSCompiler/VBCSCompiler.cs

```diff
@@ -4,13 +4,33 @@
 using System;
 using System.Collections.Specialized;
 using System.IO;
+using System.Runtime;
 
 namespace Microsoft.CodeAnalysis.CompilerServer
 {
     internal static class VBCSCompiler 
     {
         public static int Main(string[] args)
         {
+            // Enable multi-core JITing
+            // https://blogs.msdn.microsoft.com/dotnet/2012/10/18/an-easy-solution-for-improving-app-launch-performance/
+#if NET46
```

Comment:
Why are we doing this for VBCSCompiler but not the other standard EXEs?

Reply:
Indeed. Perhaps CSC should do this too. It would probably gain more from this change.